daemon/processing/block.go

```diff
@@ -24,19 +26,21 @@ var BlockLock = sync.Mutex{}
 // RunBlockProcessing runs the processing of a block at a specific height. While any height can be passed in it is
 // important to note that if the previous block is not processed it will panic to prevent corruption because blocks
 // must be processed in order.
-func RunBlockProcessing(height *uint64) uint64 {
+func RunBlockProcessing(height uint64) uint64 {
```

Comment:
Not a good way to handle this right now. Each is needed. I need to change this to output an error instead. However, that is a good amount of work since its using async channels.